packages/lib/src/vis/tiles/hooks.ts

```diff
@@ -9,23 +9,21 @@ import type { TileArray } from './models';
 import { useTooltipStore } from './store';
 
 export function useLayerScales(layerSize: Size, meshSize: Size) {
-  const { abscissaConfig, ordinateConfig } = useAxisSystemContext();
+  return useMemo(() => {
+    const xScale = createAxisScale(ScaleType.Linear, {
+      domain: [-meshSize.width / 2, meshSize.width / 2],
+      range: [0, layerSize.width],
+      clamp: true,
+    });
 
-  const xScale = createAxisScale(ScaleType.Linear, {
-    domain: [-meshSize.width / 2, meshSize.width / 2],
-    range: [0, layerSize.width],
-    reverse: abscissaConfig.flip,
-    clamp: true,
-  });
+    const yScale = createAxisScale(ScaleType.Linear, {
+      domain: [-meshSize.height / 2, meshSize.height / 2],
+      range: [0, layerSize.height],
+      clamp: true,
```

Comment:
I added a `useMemo` but the important change is elsewhere: the `reverse` args were removed: all inner components and utils from `TiledHeatmapMesh` must be oblivious to the `flip`.

Reply:
Agreed, In that case, I think it makes more sense to call `getLayerScales` into `scaleBox` as both are always called in succession.

Reply:
`meshSize` and `layerSize` should be stable, but it's no big deal anyway.